pandas/tests/io/json/test_pandas.py

```diff
@@ -810,6 +811,29 @@ def test_convert_dates(self):
         result = read_json(json, typ="series")
         tm.assert_series_equal(result, ts)
 
+    @pytest.mark.parametrize("date_format", ["epoch", "iso"])
+    @pytest.mark.parametrize("as_object", [True, False])
+    @pytest.mark.parametrize("date_typ", [datetime.datetime, pd.Timestamp])
+    def test_date_index_and_values(self, date_format, as_object, date_typ):
+        data = [date_typ(year=2020, month=1, day=1), pd.NaT]
+        if as_object:
+            data.append("a")
+
+        ser = pd.Series(data, index=data)
+        result = ser.to_json(date_format=date_format)
+
+        if date_format == "epoch":
+            expected = '{"1577836800000":1577836800000,"null":null}'
+        else:
+            expected = (
+                '{"2020-01-01T00:00:00.000Z":"2020-01-01T00:00:00.000Z"' ',"null":null}'
```

Comment:
I think CI failing on this is a false positive - @MomIsBestFriend any thoughts?

Reply:
@WillAyd I don't think the CI have a false positive. but I do want to understand what you're trying to do, are you trying ```expected``` to be a tuple, or just one string? if the answer is tuple I think you've missed a coma and the value of ```expected``` should be:

```python
expected = (
'{"2020-01-01T00:00:00.000Z":"2020-01-01T00:00:00.000Z"', ',"null":null}'
)
```

and if the value of ```expected``` spouse to be just a regular string, please just concat the two strings, so the value of ```expected``` will be:

```python
expected = (
                '{"2020-01-01T00:00:00.000Z":"2020-01-01T00:00:00.000Z" ,"null":null}'
)
```

---

And if neither of these is the answer, please explain to me what the type of ```expected``` should be, and just disable the CI. I will work on fixing it.

Reply:
Ah i see now. Thanks for confirming